ainstein_radar_drivers/src/radar_driver_o79_udp.cpp

```diff
@@ -219,15 +220,15 @@ namespace ainstein_radar_drivers
 
 	// Compute length of actual data for checking payload size:
 	msg_data_len = msg_len - RadarDriverO79UDP::msg_header_len;
-	
+
 	// Extract the target ID and data from the message:
 	int offset;
 	ainstein_radar_drivers::RadarTarget target;
 	ainstein_radar_drivers::BoundingBox box;
 	ainstein_radar_drivers::RadarTargetCartesian target_cart;
 
 	// Check the first byte for the message type ID:
-	if( buffer_[0] == RadarDriverO79UDP::msg_id_tracked_targets )
+	if( buffer_[RadarDriverO79UDP::msg_type_byte] == RadarDriverO79UDP::msg_id_tracked_targets )
```

Comment:
Was the format for UDP messages changed in 5.0.0 to move the UDP message type to the second half of the UDP message header? I don't think the Device Manual has been updated to reflect this if so. Maybe I'm missing something here, I forget all the details of 5.0.0 format changes.

Reply:
the byte order was backwards between the device manual and the source code. I changed the source code to match the manual